h/static/styles/admin.scss

```diff
@@ -31,3 +31,7 @@ body {
 .pager .pager__item--more {
   border: none;
 }
+
+.form-input__error-list {
```

Comment:
I am ambivalent about where this override is sitting but overarchingly I was Very Afraid of making changes to form CSS in a more generic place as I worried it would break user-facing forms.